packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -438,15 +469,14 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
 
   /**
    * @notice Computes a tobin tax based on the reserve ratio.
-   * @return The numerator of the tobin tax amount, where the denominator is 1000.
+   * @return The tobin tax expresesed as a fixidity fraction.
    */
   function computeTobinTax() private view returns (FixidityLib.Fraction memory) {
-    // The protocol calls for a 0.5% transfer tax on Celo Gold when the reserve ratio <= 2.
     FixidityLib.Fraction memory ratio = FixidityLib.wrap(getReserveRatio());
-    if (ratio.gte(FixidityLib.newFixed(2))) {
+    if (ratio.gte(FixidityLib.wrap(tobinTaxReserveRatio))) {
```

Comment:
any reason why `tobinTaxReserveRatio` isn't also a Fixidity frac?

Reply:
Given the timing, I tried to stay as close as possible to the previous implementation.